README.md

```diff
@@ -59,34 +36,25 @@ Looking for a professional team to build a MVM for your business on top of open-
 
 ## Customization
 
-Our project is highly customizable, [here](./doc/how_to_customize.md) is our guide on how to do it.
+This project is highly customizable, [here](./doc/how_to_customize.md) is our guide on how to do it.
 
 ## Contribution
 
 Every contribution is meaningful, kudos to everyone who helped us with developing this project! Issues and PRs are welcomed.
 
 ## We are here to help
 
-This **open-source project was developed to help the Sylius community**. If you have any additional questions, would like help with installing or configuring the plugin, or need any assistance with your Sylius project - let us know! For community support, join the official [OpenMarketplace Community Slack](https://join.slack.com/t/openmarketplacegroup/shared_invite/zt-1vejiwrbn-XZkLwRH5L0s4L9~qfkcP~g).
-
-If you want to participate in the development, you can do it by submitting [pull requests](https://github.com/BitBagCommerce/OpenMarketplace/pulls) or reporting [issues](https://github.com/BitBagCommerce/OpenMarketplace/issues).
-
-## About BitBag
-
-BitBag is a Software House working on digital commerce projects on top of best open-source technologies, such as Sylius, Shopware, Pimcore, Symfony and Vue Storefront. We work with worldwide companies who see eCommerce as an important factor of their strategy.
+This **open-source project was developed to help the Sylius community**.
+We have no dedicated support channel (yet?), but for community support, feel free to join the official [Sylius Community Slack](https://sylius.com/slack) and ask your questions in **#support** channel there.
 
-If you think we could help your business within the above-mentioned technology stack, contact us directly. We could be the right people to do the job. Fill the form on [this site](https://bitbag.io/contact-us/) or send us an e-mail at hello@bitbag.io.
+If you want to participate in the development, you can do it by submitting [pull requests](./pulls) or reporting [issues](./issues).
 
 ## License
 
 This project's source code is completely free and released under the terms of the MIT license.
 
-## Authors
+## Credits & Authors
 
-See the full list of contributors [here](https://github.com/BitBagCommerce/OpenMarketplace/contributors).
-
-## Contact
+Open Marketplace was originally developed and open-sourced by [BitBag](https://bitbag.io/), a Software House from Poland.
 
-You can contact us using the contact form on [our website](https://bitbag.io/contact-us/) or send us an e-mail to hello@bitbag.io with your question(s). We are also active on the [community Slack](https://join.slack.com/t/openmarketplacegroup/shared_invite/zt-1ij1t41wx-HfAR6~URm3OAcqm0jc423Q).
-
-[![](https://bitbag.io/wp-content/uploads/2021/08/badges-bitbag.png)](https://bitbag.io/contact-us/)
+See the full list of contributors [here](https://github.com/BitBagCommerce/OpenMarketplace/contributors).
```

Comment:
```suggestion
See the full list of contributors [here](https://github.com/rimas-kudelis/open-marketplace/graphs/contributors).
```

Reply:
Do we want to change all these URLs, knowing that we'll have to change them again later?

Reply:
@rimas-kudelis I would leave all urls until the repository name changes.